app/controllers/sugar/Actions.scala

```diff
@@ -298,8 +305,14 @@ trait Actions extends Calls with ActionHelpers {
     def refine[A](request: AuthRequest[A]) = Future.successful {
       Actions.this.organizations.withName(organization)
         .map(new AuthedOrganizationRequest[A](_, request))
-        .toRight(NotFound)
+        .toRight(notFound0(request))
     }
   }
 
+  // Convert explicit parameter to implicit
+  private def notFound0(request: Request[_]) = {
```

Comment:
This is not necessary, you can actually pass implicit parameters explicitly.

```scala
notFound()(request);
```

will work fine.